### PR TITLE
Fix issue 173 - Get server:run to work and update readme

### PR DESCRIPTION
**Pull request by jordanweiler**
This PR addresses issue https://github.com/zzorba/marvelsdb/issues/173. The `server:run` command wouldn't work until I added an additional Symfony bundle. I found a similar issue mentioned in the arkhamdb repo, which this repo is cloned from.

I also updated the README to clarify that composer 1.10 is needed for this to run since the code doesn't work with composer 2+ yet.

With these changes, I'm able to run `php bin/console server:run` and successfully load the website running at http://127.0.0.1:8000.

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -26,6 +26,7 @@ public function registerBundles()
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
+            new Symfony\Bundle\WebServerBundle\WebServerBundle(),
```
Comment (gaetan-petit) [inline app/AppKernel.php:29]: You should register the server bundle only for dev environment. You should move it after line 41.

**Comment by jordanweiler** [comment]
@Kamalisk Any chance you could take a look at this and give feedback if you have concerns? The server doesn't start locally for me without this one line in AppKernal.php and I have to keep jumping branches and adding this line in to do local development on fixes/features.

**Review by gaetan-petit — APPROVED** [review]

**Comment by clarle** [comment]
Just wanted to say this was awesome, and I was also able to get it work with this.  @Kamalisk, would be great if this could be merged, since more people could help contribute to ArkhamDB with this.

**Comment by Kamalisk** [comment]
fyi, the reason I never merged this was because I was reworking on upgrading to symfony 5, which basically means breaking everything. Since its taking me longer than I expected, I should just merge this anyway.

**Comment by gaetan-petit** [comment]
> fyi, the reason I never merged this was because I was reworking on upgrading to symfony 5, which basically means breaking everything. Since its taking me longer than I expected, I should just merge this anyway.

Nice to hear, let us know if you need any help with the upgrade. I have done a couple of upgrades like that before.

**Pull request merged**